tests/emm/services/test_git_service.py

```diff
@@ -224,6 +224,31 @@ def test_merge_base_with_directory_should_switch_directories(
         local_mock.cwd.assert_called_with(path)
 
 
+class TestCommitAll:
+    @patch(ns("local"))
+    def test_commit_all_with_no_directory_should_return_commit_all(
+        self, local_mock, git_service, mock_git
+    ):
+        mock_git.__getitem__.return_value.return_value = "abc123\n"
+
+        git_commit_all = git_service.commit_all("commit1")
+        mock_git.assert_git_call(["commit", "-am", "commit1"])
```

Comment:
Since the commit message will likely be multiple space-separated words in practice, we should probably use multiple space-separated words in the tests.